app/views/teams/show.html.erb

```diff
@@ -14,6 +14,7 @@
 
 <div class="page-header">
   <h1><%= t '.title', :default => model_class.model_name.human.titleize %></h1>
+  <%= image_tag @team.avatar_url || image_path('missing_avatar.png'), id: "preview", style: "width:350px; height:350px"%>
```

Comment:
Would it be better here to use `max-width` or `max-height` to preserve the aspect ratio of the image?

Reply:
The issue says: "The picture should be resized to squared form 350 x 350. " Sounds to me like it should be stretched to that form.

Reply:
@svitlanakalancha What do you think about resizing the image? In my opinion stretching the image is not the best solution.